lib/.changeset/perfect-mirrors-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-zod-client": patch
+---
+
+Handle hyphenated path parameters on endpoint defintion
```

Comment:
you can move this one in the root `.changeset` folder

Reply:
Sorry, I was in lib when I ran changeset. Should be all good now